test/integration/integration_test.go

```diff
@@ -235,11 +261,11 @@ func TestMultiNodeMemcache(t *testing.T) {
 
 func testBasicConfigAuthTLS(perSecond bool, local_cache_size int) func(*testing.T) {
 	s := makeSimpleRedisSettings(16381, 16382, perSecond, local_cache_size)
-	s.RedisTlsConfig = nil
 	s.RedisAuth = "password123"
 	s.RedisTls = true
 	s.RedisPerSecondAuth = "password123"
 	s.RedisPerSecondTls = true
+	s.RedisTlsCACerts = "/usr/local/share/ca-certificates/redis-stunnel.crt"
```

Comment:
Keeps all the existing successful connection unit tests working due to us removing the cert from the system level trust store.